### PR TITLE
refactor(users): move to clap and simplify the code a bit

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

File: src/uu/users/src/users.rs [hunk 1]
```diff
@@ -10,59 +10,44 @@
...
+#[macro_use]
 extern crate uucore;
 
 use uucore::utmpx::*;
 
-use getopts::Options;
+use clap::{App, Arg};
 
-static NAME: &str = "users";
+static ABOUT: &str = "Output who is currently logged in according to FILE.";
```
Comment (rivy) [inline src/uu/users/src/users.rs:21]: Nit; attempting more consistency between uutils and slightly better? grammar/clarity.

```suggestion
static ABOUT: &str = "Display who is currently logged in, according to FILE.";
```

File: src/uu/users/src/users.rs [hunk 2]
```diff
@@ -10,59 +10,44 @@
...
-        return 0;
-    }
+static ARG_FILES: &str = "files";
 
-    if matches.opt_present("version") {
-        println!("{} {}", NAME, VERSION);
-        return 0;
-    }
+fn get_usage() -> String {
+    format!("{0} [FILE]...", executable!())
```
Comment (rivy) [inline src/uu/users/src/users.rs:27]:
```suggestion
    format!("{0} [FILE]", executable!())
```
Only one is allowed.

**Review by rivy — CHANGES_REQUESTED** [review]
Nits, otherwise 👍 .

**Comment by sylvestre** [comment]
I am planning to land this Thursday 5st if no objection!

**Review by rivy — APPROVED** [review]

**Pull request merged**